org.alloytools.alloy.api/model/alloy-api.them

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
```

Comment:
`them` is a weird extension for an alloy theme file.  I suggest renaming it to either `.xml` or `.thm` or `.theme`

Reply:
This is done by Alloy? I've just saved it as is.

Reply:
This was old, removed the model directory. the default extension is `.thm` it seems